### DISCUSSION TITLE
Character creation question - Talents

### DISCUSSION THREAD

**Discussion by JeremyTHolland** [DavidRieman/WheelMUD#50] [Q&A] [answered]
How does one use the View command with a multi word talent?

I tried single and double quotes, and making the talent name one word, but no luck.

We might want to add a little help text describing how to make it work for that case.

**Comment by DavidRieman** [comment, marked as answer]
Pretty sure it's just broken.  e.g. see `this.ViewTalentDescription(commandParts[1]);` (passes just one word after view) and the implementation just looks for case insensitive exact match to that word.  Feel free to either fix or open an Issue for this.

This discussion spawned Issue #55
